tapir-cli.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
would it be possible to move this file to some subdirectory?

Reply:
I would rather submit it to a Coursier 'contrib' repo, as that would allow to install the CLI with a simple `cs fetch tapir-cli`. Just put it here for now so it could be reviewed.

Does the name (`tapir-cli`) chosen for the runnable sound good? Or would you rather call it something more specific like `tapir-codegen` or anything in those lines?

Reply:
Ah ok I see. As for naming - I think `tapir-codegen` expresses the intention better than `tapir-cli`